public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,30 @@
+// Scripts for firebase and firebase messaging
+importScripts('https://www.gstatic.com/firebasejs/8.2.0/firebase-app.js');
+importScripts('https://www.gstatic.com/firebasejs/8.2.0/firebase-messaging.js');
+
+// Initialize the Firebase app in the service worker by passing the generated config
+var firebaseConfig = {
+    apiKey: 'AIzaSyDEoVx6711d0M50SVUmXhejuO8shhzFC8I',

```

Comment:
Codacy has a fix for the issue: [Replace `····apiKey:·'AIzaSyDEoVx6711d0M50SVUmXhejuO8shhzFC8I'` with `··apiKey:·"AIzaSyDEoVx6711d0M50SVUmXhejuO8shhzFC8I"`](https://app.codacy.com/gh/albertnahas/Lingwars/pullRequest?prid=9567702)
```suggestion
  apiKey: "AIzaSyDEoVx6711d0M50SVUmXhejuO8shhzFC8I",
```